### PR TITLE
{bio}[foss/2022a] muMerge v1.1.0 w/ Python 3.10.4

### PR DESCRIPTION
**Pull request by sassy-crick**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/m/muMerge/muMerge-1.1.0-foss-2022a.eb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+name = 'muMerge'
+version = '1.1.0'
+
+homepage = 'https://github.com/Dowell-Lab/mumerge'
+description = """muMerge is a tool for combining bed regions from multiple bed files that overlap."""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+builddependencies = [('binutils', '2.38')]
```
Comment (verdurin) [inline easybuild/easyconfigs/m/muMerge/muMerge-1.1.0-foss-2022a.eb:13]: Hmm, you would only need this for a `GCCcore` toolchain?

Comment (sassy-crick) [reply easybuild/easyconfigs/m/muMerge/muMerge-1.1.0-foss-2022a.eb:13]: Thanks for finding that, I removed the `binutils`.

**Comment by akesandgren** [comment]
@boegelbot Please test @ generoso

**Review by akesandgren — APPROVED** [review]
LGTM

**Comment by akesandgren** [comment]
Going in, thanks @sassy-crick!

**Pull request merged**